changelog.d/20220602_220800_LeiGlobus_flow_run_dry_run_path_sc_16137.rst

```diff
@@ -0,0 +1,9 @@
+.. A new scriv changelog fragment.
+..
+.. Bugfixes
+.. --------
+-   `[sc-] <https://app.shortcut.com/globus/story/13753>`_
+    Prevent the loss of existing run logs when a Flow definition is updated.
```

Comment:
This fragment seems related to a different issue?

Reply:
Copy/paste/unsaved error somehow when rebasing

Reply:
The URL has been updated, but the `[sc-]` bit needs to change to `[sc-16137]`.